src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,37 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    static final int LOWEST_AGE_THRESHOLD = 18;
+    static final int HIGHEST_AGE_THRESHOLD = 125;
+    static final int SHORTEST_PASSWORD_LENGTH = 6;
+    private StorageDao storageDao;
 
     @Override
     public User register(User user) {
-        return null;
+        Integer age = user.getAge();
+        String password = user.getPassword();
```

Comment:
in my opinion, these variables are redundant

Reply:
I don't think they're redundant: otherwise we'll have to call the getAge() and getPassword() methods several times.